ruma-identifiers-validation/src/mxc.rs

```diff
@@ -0,0 +1,24 @@
+use crate::{server_name, Error};
+
+const PROTOCOL: &str = "mxc://";
+
+pub fn validate(uri: &str) -> Result<(&str, &str), Error> {
+    if let Some(prefix_index) = uri.find(PROTOCOL) {
```

Comment:
I'd prefer if this used early returns for the error cases to reduce indentation. Also I just increased the MSRV to 1.45 so you can use `strip_prefix` for this.

Reply:
Oh no, i removed it, since it wasn't available in CI. I will add it back.

Reply:
done.